wagtail/admin/templates/wagtailadmin/pages/confirm_delete.html

```diff
@@ -9,11 +9,29 @@
     <div class="nice-padding">
         <p>
             {% trans 'Are you sure you want to delete this page?' %}
-            {% if descendant_count %}
+
+            {% if descendant_count and translation_count %}
+                {% blocktrans count counter=descendant_count %}
+                    Deleting this page will also delete {{ descendant_count }} of their child pages,
+                {% plural %}
+                    Deleting this page will also delete {{ descendant_count }} more child pages,
```

Comment:
breaking the sentence on a comma like this breaks the translator context.

What about:

```jinja
{% blocktrans trimmed count counter=translation_count %}
Deleting this page will also delete {{ descendant_count }} more child pages, {{ translation_count }} translation of this page and their combined {{ combined_subpages }} translated child pages.
{% plural %}
Deleting this page will also delete {{ descendant_count }} more child pages, {{ translation_count }} translations of this page and their combined {{ combined_subpages }} translated child pages.
{% endblocktrans %}
```

Reply:
Hmm.. Good point, I didn't think about translation context. Yeah let's do what you suggested, that seems like the easiest way forward here.